examples/trip_planner/use_orchestrator.py

```diff
@@ -126,7 +124,7 @@ def inference(input_data: dict, model_type="gpt", model_name="gpt-4o-mini", use_
         "dates": user_dates,
         "interests": user_interests,
     }
-    content = inference(input_data)["output"]["content"]
```

Comment:
`inference` return dict. `write` will raise error. 

You can change in inference function from to `result.output[adaptive_orchestrator.id] ` -> `result.output[adaptive_orchestrator.id]["output"]["content"]`